public/js/logout.js

```diff
@@ -5,8 +5,10 @@ const logout = async () => {
   });
 
   if (response.ok) {
+    console.log("Successfully logged out, " + username)
```

Comment:
username is never defined, this console.log() will throw an error. I would recommend either remove the log altogether or use some exception handling. Normal users can't see the log anyway